tests/test_discover_api.py

```diff
@@ -1004,6 +1052,97 @@ def test_web_vitals_histogram_function(self) -> None:
         assert response.status_code == 200, response.data
         assert len(data["data"]) == 0, data
 
+    def test_span_op_breakdown_histogram_function(self) -> None:
+        response = self.post(
+            json.dumps(
```

Comment:
This query was generated from a real Snuba query from my testing ingesting events (end to end) locally and producing aggregated/histogram values on the product side (i.e. frontend).